bazaarvoice/emodb
# Emodbnew

Jira Issue
https://bazaarvoice.atlassian.net/browse/PD-211835

What Are We Doing Here?
Emodb is one of the 4 repositories where the critical vulnerabilities needs to be remediated in synk. Update databind version 2.10.5

DOD:
All the remediation code should be tested, reviewed and merge to mainline.
Push to QA environment and validate.

Acceptance Criteria:
All the Improper Input Validation Vulnerabilities should be remediated in Synk(11 in number).

How to Test and Verify

Check out local build :
Unit testing : mvn clean install
![image](https://github.com/bazaarvoice/emodb/assets/134580247/f0b7e75c-2072-4983-8d02-08a73977db85)


Risk
Level : Low

Required Testing
Manual.

Risk Summary
Since risk level will be low,we will be monitoring for vulnerability issues of jackson library in snyk dashboard.
